lib/tasks/peoplefinder/data.rake

```diff
@@ -77,11 +77,12 @@ namespace :peoplefinder do
         given_name = name.split.first
         surname = name.split.second
         email = "#{given_name.downcase}.#{surname.downcase}@#{DOMAIN}"
-        person = Person.find_or_create_by!(given_name: given_name, surname: surname, email: email)
-        Membership.create(person_id: person.id, group_id: demo_groups.sample.first.id) if person.memberships.empty?
+        Person.find_or_initialize_by(given_name: given_name, surname: surname, email: email).tap do |person|
+          person.memberships << Membership.new(person_id: person.id, group_id: demo_groups.sample.first.id) if person.memberships.empty?
+          person.description = 'PA to Steve Richards' if email == "personal.assistant@#{DOMAIN}"
```

Comment:
This task has been broken since a validation went in to the app over a month ago now. it also broke the db:reload task which calls it. It is for creating data in dev or elsewhere for manual testing of an elastic search scenario with similar names (which is painful to automate testing reliably and was useful for demos etc.) oh, and Steve has plenty of PAs in real life I am sure but this only creates one for the person with the relevant email address!